service/src/main/java/uk/gov/hmcts/reform/fpl/utils/PbaNumberHelper.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import lombok.NoArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.model.Applicant;
+import uk.gov.hmcts.reform.fpl.model.ApplicantParty;
+import uk.gov.hmcts.reform.fpl.model.common.C2DocumentBundle;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+@NoArgsConstructor
+@Component
+public class PbaNumberHelper {
```

Comment:
Does this need to be component? Can it not just be a set of static classes?

Reply:
not really, changed methods back to static